students/example2.html

```diff
@@ -4,14 +4,14 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="">
-    <title>Example2 Student</title>
+    <title>Jay Forbes</title>
 </head>
 <body>
-    <h1>Example2 Student</h1>
+    <h1>Jay Forbes</h1>
     <p>Anything you'd like to share / post</p>
     <ul>
-      <li><a href="http://linkedin.com">Perhaps Link to LinkedIn profile</a></li>
-      <li><a href="http://github.com">Or a link to your github profile</a></li>
+      <li><a href="www.linkedin.com/in/jay-forbes-292383204">LinkedIn profile</a></li>
+      <li><a href="https://github.com/jaygforbes">github profile</a></li>
     </ul>
     <a href="./index.html">Back to Students</a> | <a href="../index.html">Back to Main</a>
 </body>
```

Comment:
looks like you edited the example files, make sure to clean up by adding a new student file with your name jay.html and a link to this file.  update main into this branch as well.